mappings/net/minecraft/world/chunk/ProtoChunk.mapping

```diff
@@ -1,4 +1,5 @@
 CLASS net/minecraft/class_2839 net/minecraft/world/chunk/ProtoChunk
+	COMMENT
```

Comment:
```suggestion
```

Reply:
Hmm, I thought I purged all the orphaned comments